tests/test_tacd.py

```diff
@@ -281,6 +281,48 @@ def test_tacd_dut_power_switchable(strategy, online):
     assert -0.5 < r.json()["value"] < 0.5
 
 
+@pytest.mark.lg_feature("eet")
+def test_tacd_dut_power_off_floating(strategy, online):
+    """
+    Test if the tacd handles Off and OffFloating correctly.
+
+    This is done by connecting a 5V source with a 1k internal resistance to the output of the power switch.
+    During `OffFloting` the voltage must be higher as in `Off` since the additional load must be missing.
+
+    The 5V are provided from the VBus of the test-systems USB and are known to be unstable.
+    """
+
+    # Switch power switch to off. The output is loaded with 10k
+    r = requests.put(f"http://{strategy.network.address}/v1/dut/powered", data=b'"Off"')
+    assert r.status_code == 204
+
+    # Connect 5V via 1K Ohm to PWR_OUT
+    strategy.eet.link("5V_1K -> 5V -> BUS1 -> VOLT, PWR_OUT -> BUS2 -> VOLT")
```

Comment:
Shouldn't this be reversed in a finally clause?

Reply:
Same here. Since there are multiple tests that need the *eet* set to no connection I've moved this into a new fixture.